lib/meadow/data/preservation_check_writer.ex

```diff
@@ -171,11 +171,13 @@ defmodule Meadow.Data.PreservationCheckWriter do
     end
   end
 
-  defp pyramid_uri_for(file_set_id) do
+  defp pyramid_uri_for(%{role: %{id: "P"}}), do: ""
```

Comment:
👏 👏

Reply:
(This one isn't a change request. I just love seeing pattern-match solutions to issues like this.)